app/src/test/java/org/mozilla/fenix/components/toolbar/DefaultBrowserToolbarControllerTest.kt

```diff
@@ -221,20 +221,30 @@ class DefaultBrowserToolbarControllerTest {
         val item = ToolbarMenu.Item.Forward
 
         val controller = createController(scope = this)
-        controller.handleToolbarItemInteraction(item)
+        controller.handleToolbarItemClick(item)
 
         verify { metrics.track(Event.BrowserMenuItemTapped(Event.BrowserMenuItemTapped.Item.FORWARD)) }
         verify { sessionUseCases.goForward(currentSession) }
     }
 
+    @Test
+    fun handleToolbarForwardLongPress() = runBlockingTest {
```

Comment:
This is the only real change in this file. The rest is just caused by renaming `handleToolbarItemInteraction` to `handleToolbarItemClick` since long pressing is also an interaction but its handled in a separate method.